media-source-respec.html

```diff
@@ -757,38 +766,70 @@ <h4><dfn>Attaching to a media element</dfn></h4>
               purposes of aligning this specification with {{HTMLMediaElement}} resource loading and fetching
               algorithms, the underlying {{DedicatedWorkerGlobalScope}} {{MediaSource}} is the MediaSource
               object mentioned there, and the {{MediaSourceHandle}} object is the media provider object.
-              <p class="issue">MediaSourceHandle needs to be added to HTMLMediaElement's MediaProvider IDL typedef and
-              related text involving media provider objects.</p>
             </li>
           </ul>
 
-          <p>If the <a def-id="resource-fetch-algorithm"></a> was invoked with a media provider object that is a
-          {{MediaSource}} object, a {{MediaSourceHandle}} object or a URL record whose object is a {{MediaSource}}
-          object, then let mode be local, skip the first step in the <a def-id="resource-fetch-algorithm"></a>
-          (which may otherwise set mode to remote) and add the steps and clarifications below to the <a
-            def-id="Otherwise-mode-is-local"></a> section of the <a def-id="resource-fetch-algorithm"></a>.
-          <p class="note">The <a def-id="resource-fetch-algorithm"></a>'s first step is expected to eventually align with selecting local mode for URL records whose objects are media provider objects. The intent is that if the HTMLMediaElement's <code>src</code> attribute or selected child <code>&lt;source&gt;</code>'s <code>src</code> attribute is a <code>blob:</code> URL matching a [=MediaSource object URL=] when the respective <code>src</code> attribute was last changed, then that MediaSource object is used as the media provider object and current media resource in the local mode logic in the <a def-id="resource-fetch-algorithm"></a>. This also means that the remote mode logic that includes observance of any preload attribute is skipped when a MediaSource object is attached. Even with that eventual change to [[HTML]], the execution of the following steps at the beginning of the local mode logic is still required when the current media resource is a MediaSource object.</p>
-          <p class="note">Relative to the action which triggered the media element's resource selection algorithm, these steps are asynchronous. The resource fetch algorithm is run after the task that invoked the resource selection algorithm is allowed to continue and a stable state is reached. Implementations may delay the steps in the "<i>Otherwise</i>" clause, below, until the MediaSource object is ready for use.</p>
+          <p>If the [=resource fetch algorithm=] was invoked with a media provider object that is a {{MediaSource}}
+          object, a {{MediaSourceHandle}} object or a URL record whose object is a {{MediaSource}} object, then let mode
+            be local, skip the first step in the [=resource fetch algorithm=] (which may otherwise set mode to remote)
+            and add the steps and clarifications below to the <a def-id="Otherwise-mode-is-local"></a> section of the
```

Comment:
Perhaps instead of "add the steps and clarifications below to" we could say "execute the steps below at", since adding steps to an algorithm is an operation performed on text or code and here we want the steps to be executed.

Is "at" the Otherwise-mode-is-local point precise ? Should it say "immediately before" or "immediately after" the steps of the Otherwise-mode-is-local section ?

Reply:
Done. Note, doing this involved a little duplication of the "If the ... was invoked with a ... (media source / media source handle) ..." text.

Reply:
(Done in d60266dd29392fb2a7823d582515c08ff94668f8)